### ISSUE TITLE
[Bug]: Eagle static tree drafting raise Exception when the depth of tree is 1

### ISSUE THREAD

**Issue by XiaoXuan42** [NVIDIA/TensorRT-LLM#10323]
### System Info

tensorrt version: 1.2.0rc6

### Information

- [ ] The official example scripts
- [x] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [x] My own task or dataset (give details below)

### Reproduction

I run the benchmark tool:
CUDA_VISIBLE_DEVICES=0 trtllm-bench --model mbox_qwen_3b_2_raw \
  --model_path mbox_qwen_3b_2_raw \
  throughput \
  --dataset throughput_dataset_39.jsonl \
  --backend pytorch \
  --num_requests 500 \
  --max_batch_size 1 \
  --extra_llm_api_options eagle_draft_tree_one_layer.yml

eagle_draft_tree_one_layer.yml:
enable_attention_dp: false
enable_chunked_prefill: false
speculative_config:
  max_draft_len: 1
  decoding_type: Eagle
  speculative_model_dir: /export/chenxuan-trt-llm-120rc1/trt-deploy-120rc1/through_benchmark/draft_path
  eagle_choices: [[0], [1], [2], [3]]
  eagle3_one_model: false
  use_dynamic_tree: false
disable_overlap_scheduler: true
max_seq_len: 2048

### Expected behavior

The program runs without exception.

### actual behavior

I meet the following exception:
<img width="1271" height="585" alt="Image" src="https://github.com/user-attachments/assets/d23a5fd9-3f3c-4787-8c60-557f7ca1b5ce" />

### additional notes

After looking up the code of drafting_loops.py TreeDraftingLoopWrapper::forward, I found that when self.max_draft_len == 1(our case), the return_draft_logits is None which finally caused the exception.

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and checked the [documentation](https://nvidia.github.io/TensorRT-LLM/) and [examples](https://github.com/NVIDIA/TensorRT-LLM/tree/main/examples) for answers to frequently asked questions.

**Comment by mikeiovine** [comment]
Hey, check out https://github.com/NVIDIA/TensorRT-LLM/pull/10385 for a fix. Note that the trees are still an experimental/new feature so there may be more problems (the overlap scheduler seems to have some issues right now, for instance).
